bin/verilator

```diff
@@ -4368,6 +4404,14 @@ Ignoring this warning may make Verilator simulations differ from other
 simulators, if the increased precision of real affects your model or DPI
 calls.
 
+=item SPLITVAR
+
+Warns that a variable with split_var metacomment was not split.
+Possible reasons are
+
+  The datatype is not supported for splitting. (e.g. real)
+  The access pattern of the variable can not be determined statically. (e.g. memory)
```

Comment:
Warns that a variable with the split_var metacomment was not split as
requested.  Possible reasons include:

* The datatype of the variable is not supported for splitting. (e.g. is a
real).

* The access pattern of the variable can not be determined
statically. (e.g. is accessed as a memory).